docs/en/week08/08-3.md

```diff
@@ -9,57 +9,57 @@ date: 25 March 2021
 
 ## Recap: Predictive Latent Variable Model
 
-As we know latent variables are useful in providing additional information that help in predicting multiple values of the target $\boldsymbol{y}$, associated with the input $x$. We have seen earlier, in order to predict the target variable $\boldsymbol{y}$ from latent variable $z$, the energy function is given as,
+As we know latent variables are useful in providing additional information that help in predicting multiple values of the target $\vy$, associated with the input $x$. We have seen earlier, in order to predict the target variable $\vy$ from latent variable $z$, the energy function is given as,
 
 $$
-E(\boldsymbol{y},z) = [y_1 - g_1(z)]^2 + [y_2 - g_2(z)]^2, \boldsymbol{y} \in \boldsymbol{Y}
+\red{E}(\vy,\vz) = [\vy{_1} - g_1(\vz)]^2 + [y_2 - g_2(\vz)]^2, \vy \in \boldsymbol{Y}
 $$
 
 To be clear, we show the unit of $\boldsymbol{g}$:
 
 $$
 \boldsymbol{g} = [g_{1}, g_{2}]^\top, \mathbb{R} \rightarrow \mathbb{R}^{2} \\
-\boldsymbol{g}(z) = [w_{1}\cos(z), w_{2}\sin(z)]
+\boldsymbol{g}(z) = [w_{1}\cos(\vz), w_{2}\sin(\vz)]
 $$
 
 In this case, the latent variable $z$ is one dimensional. However, if the $z$ has a higher dimension, this could lead to an energy function of $0$  everywhere. Hence, in order to ensure lower energy values for more compatible predictions we introduce a regularized factor.
```

Comment:
From here on `z` is actually a vector, so it should be replaced with `\vz`.
Plz, notice the difference. It's relevant.